man/data.table.Rd

```diff
@@ -25,7 +25,7 @@ data.table(\dots, keep.rownames=FALSE, check.names=FALSE, key=NULL, stringsAsFac
   mult = "all",
   roll = FALSE,
   rollends = if (roll=="nearest") c(TRUE,TRUE)
-             else if (roll>=0) c(FALSE,TRUE)
+             else if (roll>=0L) c(FALSE,TRUE)
```

Comment:
not sure about this one, `roll` is more likely to be numeric than integer (Inf) so leaving it as numeric make sense.

Reply:
I'm assuming real>int is the same speed as real>real (so that here we simply save the memory of creating numeric[1] every time [ is called) but maybe that's wrong.

or you're thinking that this might signal the wrong thing to users (roll compared to integer so maybe roll has to be integer)?

Reply:
Yeah I'd side with Jan and leave it as 0.  There's a force(rollends) in `[.data.table` and just above that, roll is made to be `as.double`.  But it's more about not putting a potentially confusing "L" in the argument default which is visible in the manual page and args(`[.data.table`).  (One less thing out of many for users to be confused about.)

Reply:
ok works for me, just add a note in the grep cmd I guess